src/test/java/api/items/ItemApiMoveExamples.java

```diff
@@ -0,0 +1,148 @@
+
+    final UUID existedHoldingId = createHoldingForInstance(instanceId);
+    final UUID newHoldingId = createHoldingForInstance(instanceId);
+
+    Assert.assertNotEquals(existedHoldingId, newHoldingId);
+
+    final UUID nonExistedItemId = UUID.randomUUID();
+
+    final IndividualResource createItem1 = itemsClient.create(new ItemRequestBuilder().forHolding(existedHoldingId)
+      .withBarcode("645398607547")
+      .withStatus(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject itemsMoveRequestBody = new ItemsMoveRequestBuilder(newHoldingId,
+        new JsonArray(Arrays.asList(createItem1.getId(), nonExistedItemId))).create();
+
+    CompletableFuture<Response> postItemsMoveCompleted = new CompletableFuture<>();
+    okapiClient.post(ApiRoot.move(), itemsMoveRequestBody, ResponseHandler.any(postItemsMoveCompleted));
+    Response postItemsMoveResponse = postItemsMoveCompleted.get(5, TimeUnit.SECONDS);
+
+    assertThat(postItemsMoveResponse.getStatusCode(), is(404));
+    assertThat(postItemsMoveResponse.getContentType(), containsString(APPLICATION_JSON));
+
+    List notFoundIds = postItemsMoveResponse.getJson()
+      .getJsonObject("errors")
+      .getJsonArray("ids")
+      .getList();
+    assertThat(notFoundIds.size(), is(1));
+    assertThat(notFoundIds.get(0), equalTo(nonExistedItemId.toString()));
+
+    JsonObject updatedItem1 = itemsClient.getById(createItem1.getId())
+      .getJson();
+    assertThat(newHoldingId.toString(), equalTo(updatedItem1.getString(HOLDINGS_RECORD_ID)));
+  }
+
+  @Test
+  public void cannotCreateMoveWithoutToHoldingsRecordId()
+      throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+    JsonObject itemMoveWithoutToHoldingsRecordId = new ItemsMoveRequestBuilder(null,
+        new JsonArray(Collections.singletonList(UUID.randomUUID()))).create();
+    okapiClient.post(ApiRoot.move(), itemMoveWithoutToHoldingsRecordId, ResponseHandler.any(postCompleted));
+    Response postResponse1 = postCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(postResponse1.getStatusCode(), is(422));
+    assertThat(postResponse1.getContentType(), containsString(APPLICATION_JSON));
+  }
+
+  @Test
+  public void cannotCreateMoveWithoutItemIds()
+      throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+    JsonObject itemMoveWithoutItemIds = new ItemsMoveRequestBuilder(UUID.randomUUID(), new JsonArray()).create();
+    okapiClient.post(ApiRoot.move(), itemMoveWithoutItemIds, ResponseHandler.any(postCompleted));
+    Response postResponse2 = postCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(postResponse2.getStatusCode(), is(422));
+    assertThat(postResponse2.getContentType(), containsString(APPLICATION_JSON));
```

Comment:
Similarly to the above, please also check the specific error is included in the response and I think it would be easier to follow this test if the steps were separated by some whitespace